packages/core/src/Stores/traders-hub-store.js

```diff
@@ -555,7 +579,9 @@ export default class TradersHubStore extends BaseStore {
         const existing_accounts = current_list_keys
             .filter(key => {
                 const maltainvest_account = current_list[key].landing_company_short === 'maltainvest';
-                if (
+                if (product === 'stp') {
```

Comment:
Suggestion - best not to hardcode strings in the code.

There are multiple instance of stp string being hardcoded in this PR.